homeassistant/components/onvif/parsers.py

```diff
@@ -214,6 +214,35 @@ async def async_parse_cell_motion_detector(uid: str, msg) -> Event:
         return None
 
 
+@PARSERS.register("tns1:RuleEngine/LineDetector/Crossed")
+# pylint: disable=protected-access
+async def async_parse_line_crossed(uid: str, msg) -> Event:
+    """Handle parsing event message.
+
+    Topic: tns1:RuleEngine/LineDetector/Crossed
+    """
+    try:
+        video_source = ""
+        video_analytics = ""
+        rule = ""
+        for source in msg.Message._value_1.Source.SimpleItem:
```

Comment:
Is there some way to not have to access the private attribute?

Reply:
Unfortunately not.  This is a dynamic class generated by `zeep` and that is how it handles nested values: https://docs.python-zeep.org/en/master/datastructures.html#nested-using-value-1